.github/workflows/deploy.yml

```diff
@@ -0,0 +1,86 @@
+name: Deploy
+
+on:
+  push:
+    tags: 
+      - '*'
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
```

Comment:
what about using v2 for this checkout?

Reply:
wow, didn't knew there is `v2` avalable ;-)